dandiapi/api/tests/test_zarr.py

```diff
@@ -48,8 +49,8 @@ def test_zarr_rest_get(
 
 @pytest.mark.django_db
 def test_zarr_rest_get_very_big(authenticated_api_client, zarr_archive_factory):
-    ten_quadrillion = 10**16
-    ten_petabytes = 10**16
+    ten_quadrillion = 10 ** 16
+    ten_petabytes = 10 ** 16
```

Comment:
just a note that black just changed their styling to drop the spaces around this (i.e. what it looked like before the change). haven't followed the social world about the reaction.

Reply:
My bad, I must be running an old version of black.